tests/upgrades/test_virtwho.py

```diff
@@ -48,17 +48,15 @@ class scenario_positive_virt_who(APITestCase):
     """
 
     @classmethod
-    @skip_if_not_set('virtwho')
     def setUpClass(cls):
         cls.org_name = 'virtwho_upgrade_org_name'
         cls.satellite_url = settings.server.hostname
-        cls.hypervisor_type = settings.virtwho.hypervisor_type
-        cls.hypervisor_server = settings.virtwho.hypervisor_server
-        cls.hypervisor_username = settings.virtwho.hypervisor_username
-        cls.hypervisor_password = settings.virtwho.hypervisor_password
-        cls.hypervisor_config_file = settings.virtwho.hypervisor_config_file
-        cls.vdc_physical = settings.virtwho.sku_vdc_physical
-        cls.vdc_virtual = settings.virtwho.sku_vdc_virtual
+        cls.hypervisor_type = virtwho.esx.hypervisor_type
```

Comment:
So, this will be run for ESX and not any other hypervisor? Is that intended? I think ideally we should run this test for all supported hypervisor types.

Reply:
Thanks for bringing this up, we have considered this. For virt-who plugin upgrade testing, we only test the common scenarios, it means the test cases have the same steps for all the hypervisors. So, we think only for ESX will be OK.